packages/app-project/pages/404.js

```diff
@@ -0,0 +1,74 @@
+import styled from 'styled-components'
+
+// NOTE: For static compiled 404 this gets run at compile time instead of on every request
+// This means every deploy will introduce the randomness of the 404
+let randomImage = Math.round(Math.random() * 8) + 1 // 1-9
+
+const Container = styled.div`
+  text-align: center;
+  position: absolute;
+  width: 100%;
+  height: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  color: white;
+  background-color: #1E1E1E;
+  z-index: -2;
+  
+  div.image {
+    position: fixed;
+    top: 0;
+    left: 0;
+    right: 0;
+    bottom: 0;
+
+    background-image: url("/images/background${randomImage}.png");
```

Comment:
```suggestion
    background-image: url("/projects/assets/background${randomImage}.png");
```


Related to my comments about our proxy setup for the NextJS apps. `https://www.zooniverse.org/Images` will try to load the image from PFE. 

Static images for the project app are in `public/assets`, and the URL is `/projects/assets`.

Reply:
Incorporated